portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,30 +29,48 @@
 /** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private List<String> comments = new ArrayList<>(
-    Arrays.asList("Nice!",
-                  "Very interesting.",
-                  "I would like to know more about your favourite TV shows!"));
+  private final Deque<Comment> comments = new LinkedList<>();
```

Comment:
Cool idea with the `Deque`. I recommend using the `ArrayDeque` implementation over a `LinkedList` since it is faster for almost all operations, especially when reading all values (which is internally done when you serialize it as json)

Reply:
Since the comments won't be stored in any persistent storage, consider hard-coding some test comments (as previously). It would make testing displaying much easier for you (no need to add a test comment after every restart of the server).